app/model.rb

```diff
@@ -17,15 +18,19 @@ class User
 DataMapper.finalize.auto_upgrade!
 
 class Ocr
-  def init
-    @engine = Tesseract::Engine.new {|engine|
-                engine.language = :lol # arbitrary
+  def initialize
+    @engine = ::Tesseract::Engine.new {|engine|
+                engine.language = :eng
                 engine.page_segmentation_mode = 4
-                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].join # thanks ruby, made this easy
+                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].flatten # thanks ruby, made this easy
```

Comment:
I believe this needs to be a string, rather than an arr type.
reference: https://github.com/meh/ruby-tesseract-ocr/blob/master/examples/nerdz-captcha-breaker/break.rb#L34